decidim-proposals/lib/decidim/proposals/feature.rb

```diff
@@ -37,6 +37,19 @@
     resource.template = "decidim/proposals/proposals/linked_proposals"
   end
 
+  feature.register_stat :accepted_proposals_count, primary: true do |features, start_at, end_at|
```

Comment:
Thanks @josepjaume, that's what I meant. My question was more: "Why are accepted proposals highlighted instead of all proposals?"

I understood what `primary` does from a technical PoV.